resources/configs/opensuse/gn_flags

```diff
@@ -1,7 +1,14 @@
 is_clang=false
+linux_use_bundled_binutils=false
+use_custom_libcxx=false
+use_cups=true
+is_component_build=false
 is_component_ffmpeg=true
+use_pulseaudio=true
 link_pulseaudio=true
-use_allocator="tmalloc"
+use_allocator="tcmalloc"
 use_aura=true
+use_gconf=true
 use_gtk3=true
 use_kerberos=true
+use_gold=true
```

Comment:
At this rate, perhaps it would help to make `opensuse` inherit from `linux_rooted` instead of `common`? Or are there changes in `linux_rooted` you don't want?

Reply:
They do seem to have much in common. I'll try chaning the base config to linux_rooted on the next minor version update and see how it goes.